packages/sdk-client-v3/src/types/types.d.ts

```diff
@@ -18,13 +18,13 @@ export type MiddlewareResponse<T = unknown> = {
 
 export type HttpErrorType = {
   name?: string
-  message: string
-  code?: number
+  message?: string
+  code?: string
```

Comment:
In the JIRA task this is written: `We can’t change the code field to string as it would be a breaking change.`

Reply:
That was my suggestion, I even went ahead to suggest we add a new field for it, but you specifically suggested that since it wasn't documented, we can go ahead and make the change. Have you forgotten? 🙃